closeio_api/__init__.py

```diff
@@ -47,6 +49,10 @@ def __init__(self, base_url, api_key=None, tz_offset=None, max_retries=5,
             self.session.auth = (api_key, '')
 
         self.session.headers.update({
+            'User-Agent': 'python closeio v{} {}'.format(
```

Comment:
Drop the `io`

Reply:
closeio is the name of the package in pypi, so i don't think we should drop it.

Reply:
Makes sense. Opened #105